libcpuid/libcpuid.h

```diff
@@ -1108,9 +1109,9 @@ int cpu_msrinfo(struct msr_driver_t* handle, cpu_msrinfo_request_t which);
 
 /**
  * @brief Writes the raw MSR data to a text file
- * @param data - a pointer to msr_driver_t structure
  * @param filename - the path of the file, where the serialized data should be
  *                   written. If empty, stdout will be used.
+ * @param handle -  a handle to the MSR reader driver, as created by cpu_msr_driver_open ?
```

Comment:
1. Why is there '?' at the end?
2. The parameters should be listed in the order given in the function call, not alphabetically, so you shouldn't have swapped them

Reply:
1) I wasn't 100% sure. I can remove that.
2) Fair enough I can turn that around.